MapboxDirections/MBRouteOptions.swift

```diff
@@ -391,6 +395,13 @@ open class RouteOptions: NSObject, NSSecureCoding, NSCopying{
     open var distanceMeasurementSystem: MeasurementSystem = Locale.autoupdatingCurrent.usesMetric ? .metric : .imperial
     
 
+    /**
+     An Array of `RoadClass` that the routing engine will attempt to avoid.
```

Comment:
Typo: “array”, not “Array”.

> The route classes that the calculated routes will avoid.
> 
> Currently, you can only specify a single road class to avoid.